danielSbastos/saxophone-infochart-back
# Add StructureController and its actions

Hi,
Here `StructureController` is added together with the routes for each piece of a saxophone; reed, mouthpiece, neck, bell, octave pin and neck strap. Each action returns the meaning of its respective sax piece.

## Tests

1) Spin up the server with `mix phx.server`
2) Make a get request to an endpoint, for example, `curl http://localhost:4000/api/reed/`
3) Check that the result matches the ones in the controller. With the example above, expect 
 > Responsible for generating sound when the
  player puts in his mouth and vibrates it.
